tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -89,12 +89,12 @@ Feature: antivirus
   Scenario Outline: upload a file with the virus to a public share
     Given using <dav-path-version> DAV path
     And user "Alice" has created folder "/uploadFolder"
-    And user "Alice" has created a public link share with settings
-      | path        | /uploadFolder            |
-      | name        | sharedlink               |
-      | permissions | change                   |
-      | expireDate  | 2040-01-01T23:59:59+0100 |
-      | password    | %public%                 |
+    And user "Alice" has created the following link share:
+      | resource           | uploadFolder             |
+      | space              | Personal                 |
+      | permissionsRole    | edit                     |
+      | password           | %public%                 |
+      | expirationDateTime | 2040-01-01T23:59:59.000Z |
     When user "Alice" uploads file "filesForUpload/filesWithVirus/<file-name>" to "/<new-file-name>" using the WebDAV API
     Then the HTTP status code should be "201"
```

Comment:
I doubt is these scenarios are right :thinking: 
Scenario says
`upload a file with the virus to a public share`
but we are doing
`user "Alice" uploads file "filesForUpload/filesWithVirus/<file-name>" to "/<new-file-name>" using the WebDAV API`

Reply:
Please, check these scenarios:
```feature
Scenario Outline: upload a file with the virus to a public share
Scenario Outline: upload a file with the virus to a password-protected public share
```